packages/mobile-signing-provider/esbuild.config.js

```diff
@@ -1,11 +1,467 @@
+          }));
+
+          build.onLoad(
+            { filter: /^virtual:tslib$/, namespace: 'virtual' },
+            () => ({
+              contents: `
+                export function __awaiter(thisArg, _arguments, P, generator) {
+                  return new (P || (P = Promise))(function(resolve, reject) {
+                    function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
+                    function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
+                    function step(result) { result.done ? resolve(result.value) : new P(function(resolve) { resolve(result.value); }).then(fulfilled, rejected); }
+                    step((generator = generator.apply(thisArg, _arguments || [])).next());
+                  });
+                }
+      
+                export function __exportStar(m, o) {
+                  for (var p in m) if (p !== "default" && !Object.prototype.hasOwnProperty.call(o, p)) __createBinding(o, m, p);
+                }
+      
+                export function __createBinding(o, m, k, k2) {
+                  if (k2 === undefined) k2 = k;
+                  var desc = Object.getOwnPropertyDescriptor(m, k);
+                  if (!desc || ("get" in desc ? !m.__esModule : desc.writable || desc.configurable)) {
+                    desc = { enumerable: true, get: function() { return m[k]; } };
+                  }
+                  Object.defineProperty(o, k2, desc);
+                }
+      
+                export function __values(o) {
+                  var s = typeof Symbol === "function" && Symbol.iterator, m = s && o[s], i = 0;
+                  if (m) return m.call(o);
+                  if (o && typeof o.length === "number") return {
+                    next: function () {
+                      if (o && i >= o.length) o = void 0;
+                      return { value: o && o[i++], done: !o };
+                    }
+                  };
+                  throw new TypeError(s ? "Object is not iterable." : "Symbol.iterator is not defined.");
+                }
+      
+                export function __read(o, n) {
+                  var m = typeof Symbol === "function" && o[Symbol.iterator];
+                  if (!m) return o;
+                  var i = m.call(o), r, ar = [], e;
+                  try {
+                    while ((n === void 0 || n-- > 0) && !(r = i.next()).done) ar.push(r.value);
+                  }
+                  catch (error) { e = { error: error }; }
+                  finally {
+                    try {
+                      if (r && !r.done && (m = i["return"])) m.call(i);
+                    }
+                    finally { if (e) throw e.error; }
+                  }
+                  return ar;
+                }
+              `,
+              loader: 'js',
+            })
+          );
+        },
+      },
+      {
+        name: 'replace-unstorage',
```

Comment:
not sure about this one, lib for browser so localstorage should be enough, needs more info